app.py

```diff
@@ -49,6 +51,6 @@ def bot(history):
             msg_textbox.submit(user, [msg_textbox, chatbot], [msg_textbox, chatbot], queue=False).then(
                 bot, chatbot, chatbot
             )
-
+#server_name='0.0.0.0', server_port=int(port)
```

Comment:
Do we want sharing to be on by default? Also, commented out line may be removed if unneeded